Wizardry/Core/BattleSys/Source/PreBattleCalc.c

```diff
@@ -653,10 +652,48 @@ void PreBattleCalcAuraEffect(struct BattleUnit * attacker, struct BattleUnit * d
                 if (SkillTester(unit, SID_NightTide) && range1[i] == 1)
                 {
                     attacker->battleDefense += 5;
-
                 }
 #endif
 
+#if (defined(SID_SpurStr) && (SID_SpurStr < MAX_SKILL_NUM))
+                if (SkillTester(unit, SID_SpurStr) && range1[i] == 1)
+		{
+			if(!IsMagicAttack(attacker))
+                    	attacker->battleAttack  += 4;
+		}
+#endif
+
+#if (defined(SID_SpurMag) && (SID_SpurMag < MAX_SKILL_NUM))
+                if (SkillTester(unit, SID_SpurMag) && range1[i] == 1)
+		{
+			if(IsMagicAttack(attacker))
```

Comment:
Use four space rather than one TAB on indentation.

Reply:
Should be good now. Sorry for the disagreement, I had now idea Github handled tabulation differently than 4 space